packages/remix-netlify-edge/globals.ts

```diff
@@ -0,0 +1,6 @@
+import "@remix-run/deno/globals";
+
+// Deno will exist in Deno land and we only need the minimal types for ProcessEnv
+// The types for this for Deno are defined in globals.d.ts in the Netlify template
+// @ts-ignore
+globalThis.process ||= { env: Deno.env.toObject() };
```

Comment:
Is this recommended to do in the Deno world?

If so, should we add this to the deno globals instead?
If not, we should maybe just remove this file altogether?

CC/ @pcattori

Reply:
We generally do this for frameworks tageting Netlify Edge Functions so that people following tutorials and docs elsewhere can access `process.env` as expected. Unlike someone targeting Deno directly, a person deploying a Remix site to Netlify Edge Functions probably isn't thinking of the fact they're building a Deno site, so we generally try to make things Just Work where possible.